project/AppDependencies.scala

```diff
@@ -8,14 +8,14 @@ object AppDependencies {
     ws,
     "uk.gov.hmrc"          %% "play-conditional-form-mapping" % "1.2.0-play-26",
     "uk.gov.hmrc"          %% "logback-json-logger"           % "4.6.0",
-    "uk.gov.hmrc"          %% "govuk-template"                % "5.48.0-play-26",
-    "uk.gov.hmrc"          %% "play-health"                   % "3.14.0-play-26",
-    "uk.gov.hmrc"          %% "play-ui"                       % "8.5.0-play-26",
-    "uk.gov.hmrc"          %% "bootstrap-play-26"             % "1.3.0",
-    "uk.gov.hmrc"          %% "play-frontend-govuk"           % "0.32.0-play-26",
-    "uk.gov.hmrc"          %% "auth-client"                   % "2.32.1-play-26",
+    "uk.gov.hmrc"          %% "govuk-template"                % "5.54.0-play-26",
+    "uk.gov.hmrc"          %% "play-health"                   % "3.15.0-play-26",
+    "uk.gov.hmrc"          %% "play-ui"                       % "8.9.0-play-26",
+    "uk.gov.hmrc"          %% "bootstrap-play-26"             % "1.7.0",
+    "uk.gov.hmrc"          %% "play-frontend-govuk"           % "0.43.0-play-26",
+    "uk.gov.hmrc"          %% "auth-client"                   % "3.0.0-play-27",
```

Comment:
There should be `play-26`

play-27 is for play 2.7, we're still use the 2.6

Reply:
Oops.  I blindly copied the recommendation on the catalogue page - good spot!